### DISCUSSION TITLE
Dark mode for the notebook as rendered?

### DISCUSSION THREAD

**Discussion by hopezh** [quarto-dev/quarto-nvim#68] [Ideas]
It would be great to have a `dark mode` to choose from, like that for Google Colab as shown below, so that we don't need to stare at the chart against the blinding white background of the notebook. 

![image](https://github.com/quarto-dev/quarto-nvim/assets/23435810/6861b69a-529b-43dd-8953-3667cd0fa777)

**Comment by jmbuhr** [comment]
I don't understand the question. Your neovim theme is independent of your quarto theme and your matplotlib theme etc. And the Neovim theme I have enable by default in the Kickstarter config is in fact a dark one.

**Reply by jmbuhr** [reply]
are you maybe looking for https://quarto.org/docs/output-formats/html-themes.html?

**Reply by hopezh** [reply]
Yes, exactly, I should make it clear it's about the html styling. Apologize for my lack of clarity.

**Reply by hopezh** [reply]
Here it is. The dark theme for html:

![image](https://github.com/quarto-dev/quarto-nvim/assets/23435810/21b93f10-56a3-4b32-90b8-fe3e3cb4d5da)
